pypeit/images/combineimage.py

```diff
@@ -144,32 +196,43 @@ def run(self, bias=None, flatimages=None, ignore_saturation=False, sigma_clip=Tr
             print(msgs.indent() + strout.format("Filename", "Lamp status"))
             print(msgs.indent() + '-'*maxlen + "  " + '-'*maxlmp)
             for ff, file in enumerate(self.files):
-                print(msgs.indent() + strout.format(os.path.split(file)[1], " ".join(lampstat[ff].split("_"))))
+                print(msgs.indent()
+                      + strout.format(os.path.split(file)[1], " ".join(lampstat[ff].split("_"))))
             print(msgs.indent() + '-'*maxlen + "  " + '-'*maxlmp)
 
         # Coadd them
-        weights = np.ones(nimages)/float(nimages)
-        img_list = [img_stack]
         var_stack = utils.inverse(ivar_stack)
-        var_list = [var_stack, rn2img_stack]
         if combine_method == 'weightmean':
```

Comment:
Maybe we should rename this option as just 'mean' since we are using uniform weights.

Reply:
I've left this for now, but can make this change, if all agree.